plugins/logitech-rallysystem/fu-logitech-rallysystem-audio-device.c

```diff
@@ -97,6 +100,14 @@ fu_logitech_rallysystem_audio_device_set_version(FuLogitechRallysystemAudioDevic
 		G_BIG_ENDIAN,
 		error))
 		return FALSE;
+	/*
+	 * device reports system version in 3 bytes: major.minor.build
+	 * convert major.minor.build -> major.minor.0.build
```

Comment:
Yes, FWUPD_VERSION_FORMAT_TRIPLET is valid, no other change needed.

Reply:
Usually firmware report version as Byte1=major; Byte2=minor, Byte3&4=build, but this device reports Byte1=major, Byte2=minor, Byte3=build, Byte4=Ignore. Sorry for the confusion, used "." as byte separator.

Reply:
I think a better way to do this would be to set the `fwversion` into version_raw as before, and instead construct the user-visible triplet in `fu_logitech_rallysystem_audio_device_convert_version()`